test/run_tests.py

```diff
@@ -36,9 +36,9 @@
 
 Examples:
 
-  run_tests.py python chrome
-  run_tests.py jython firefox --suite list
-  run_tests.py python chrome --scusername your_username --sckey account_key
+  run_tests.py chrome
+  run_tests.py --suite list --interpreter jython firefox
```

Comment:
Better to have --suite at end because is a list.
run_tests.py --interpreter jython firefox --suite list